### ISSUE TITLE
Securing Cookies

### ISSUE THREAD

**Issue by amokdad** [DuendeSoftware/Support#1100]
We are using denude server for our client CGB and their security team is reporting security issue related to the below cookies:

.AspNetCore.Correlation.
.AspNetCore.OpenIdConnect.

The issue is “Cookie with insecure of improper or missing samesite attribute

**Comment by josephdecock** [comment]
These cookies are issued by the Microsoft ASP.NET OpenID Connect handler. They exist to persist state that is needed during the OpenId Connect protocol flow, which by design redirects from the client application to an external identity provider. The flow both needs this state, and is designed to work with an external provider which by design results in cross-site or even cross-origin redirects, the cookies that persist that state can't be restricted to a same-site context.

To sum up, this is a false positive from the security team for a cookie that isn't actually issued by IdentityServer.

**Comment by josephdecock** [comment]
Is anything further needed on this issue, or should we close it?

**Comment by josephdecock** [comment]
Closing due to inactivity, but feel free to reopen if necessary.
